edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -78,6 +78,8 @@ def authenticate(self, request):
 
         except jwt.InvalidTokenError:
             raise exceptions.AuthenticationFailed()
+        except exceptions.NotAuthenticated:
```

Comment:
@robrap https://github.com/Styria-Digital/django-rest-framework-jwt/pull/27/files#diff-bc0105dbb10d8b2bcacf8d191d5d0195R105
This line throws error. Previously it return None.

https://github.com/Styria-Digital/django-rest-framework-jwt/issues/57

Reply:
Issue already reported. But update.